pkg/auth/tokens/token_util.go

```diff
@@ -119,17 +122,55 @@ func VerifyToken(storedToken *v3.Token, tokenName, tokenKey string) (int, error)
 // ConvertTokenKeyToHash takes a token with an un-hashed key and converts it to a hashed key
 func ConvertTokenKeyToHash(token *v3.Token) error {
 	if token != nil && len(token.Token) > 0 {
-		var err error
-		hashedToken, err := common.CreateHash(token.Token)
-		if err != nil {
-			logrus.Errorf("Failed to generate hash from token: %v", err)
-			return errors.New("failed to generate hash from token")
-		}
-		token.Token = hashedToken
+		token.Token = CreateHash(token.Token)
 		if token.Annotations == nil {
 			token.Annotations = map[string]string{}
 		}
 		token.Annotations[TokenHashed] = "true"
 	}
 	return nil
 }
+
+const hashFormat = "$%d:%s:%s" // $version:algorithm:hash -> $1:sha256:abc
+const Version = 1
+const Algorithm = "sha256"
+
+func CreateHash(secretKey string) string {
```

Comment:
I think we should document when to use this instead of the other VerifyHash. Which tokens require this vs the other